core/arch/arm/plat-marvell/sub.mk

```diff
@@ -3,3 +3,7 @@ srcs-y += main.c
 ifeq ($(PLATFORM_FLAVOR_armada7k8k),y)
 srcs-y += armada7k8k/hal_sec_perf.c
 endif
+
+ifeq ($(PLATFORM_FLAVOR_armada3700),y)
+srcs-y += armada3700/hal_sec_perf.c
+endif
```

Comment:
Better written as:

`src-$(PLATFORM_FLAVOR_armada3700) += armada3700/hal_sec_perf.c`

(same for 7k8k above BTW).

Reply:
Done